pkg/sql/parser/sql.y

```diff
@@ -1254,6 +1254,7 @@ func (u *sqlSymUnion) doBlockOption() tree.DoBlockOption {
 %type <tree.Statement> create_stmt
 %type <tree.Statement> create_schedule_stmt
 %type <tree.Statement> create_changefeed_stmt create_schedule_for_changefeed_stmt
+%type <tree.Statement> create_database_changefeed_stmt create_schedule_for_changefeed_stmt
```

Comment:
I don't think you should be duplicating `create_schedule_for_changefeed_stmt` from the line above
```suggestion
%type <tree.Statement> create_database_changefeed_stmt
```